lib/vue/src/components/input/CAceEditor.vue

```diff
@@ -115,6 +120,11 @@ export default {
 
   methods: {
     editorInit (editor) {
+      if (this.initializeEditor && typeof this.initializeEditor === 'function') {
```

Comment:
If possible the best case scenario would be to just provide the props and let the c-ace-editor do the init. We can sync on this